Localization/zh-CN/covid-19.po

```diff
@@ -0,0 +1,92 @@
+
+msgid "An experimental structure of an Ebola protein doesn’t have obvious druggable sites (no deep pockets among the atoms shown as spheres).Our simulations captured a motion that creates a potentially druggable site in this Ebola protein. Instead of showing spheres for each atom, this cartoon shows a ribbon tracing the linear chain of amino acids (chemicals) the protein is made of."
+msgstr "埃博拉病毒蛋白的实验结构没有明显的可药用位点（显示为球形的原子中没有深层的囊袋）。我们的模拟捕获了一个运动，该运动在该埃博拉病毒蛋白中创建了潜在的可药物化部位。该视频演示没有显示每个原子的球体，而是显示了一条条带，该条带描绘了蛋白质的氨基酸（化学物质）的线性链。"
+
+msgid "We want to do the same thing with coronavirus."
+msgstr "我们想在冠状病毒上采取相同的措施。"
+
+msgid "Spreading the knowledge, Opensource"
+msgstr "传播知识，并且是开源的"
+
+msgid "When we are done with our analysis we upload the data here:"
+msgstr "当我们完成分析时，会将数据上传到这里："
+
+msgid "We also make all data publicly available, so that other people working in the field can check our analysis and anyone with new methods (e.g. the always growing machine learning data analysis) can look at them at any time: [https://osf.io/2h6p4/wiki/home/](https://osf.io/2h6p4/wiki/home/) and [https://osf.io/dp4cb/wiki/home/](https://osf.io/dp4cb/wiki/home/)"
+msgstr "我们也会公开所有的数据，无论是在相同领域工作的人，还是其他使用新型方法的人（比如，愈发广泛使用的基于机器学习的数据分析），都可以在任何时候查看这些数据链接：[https://osf.io/2h6p4/wiki/home/](https://osf.io/2h6p4/wiki/home/)，[https://osf.io/dp4cb/wiki/home/](https://osf.io/dp4cb/wiki/home/)。"
+
+msgid "We are also collaborating with other labs and groups outside the Foldingathome Consortium to solve COVID-19 asap."
+msgstr "我们还与其他实验室和小组（没有参与Foling@home项目的）合作，以尽快解决COVID-19的大流行。"
+
+msgid "What you can do"
+msgstr "您能帮到我们什么"
+
+msgid "1. Donate computing power:"
+msgstr "1. 分享计算资源："
+
+msgid "[Downloading Folding@home](/start-folding/) and helping us run simulations is the primary way to contribute. These calculations are enormous and every little bit helps! Each simulation you run is like buying a lottery ticket. The more tickets we buy, the better our chances of hitting the jackpot. Usually, your computer will never be idle, but we’ve had such an enthusiastic response to our COVID-19 work that you will see some intermittent downtime as we sprint to setup more simulations. Please be patient with us! There is a lot of valuable science to be done, and we’re getting it running as quickly as we can."
+msgstr "[下载 Foling@home 模拟软件](/start-folding/)并帮助我们运行模拟是主要的贡献方式。模拟运算非常复杂并且每一点计算都可以帮到我们！您进行的每次模拟都像买彩票一样。我们购买的门票越多，我们赢得大奖的机会就越大。通常情况下，您的计算机永远不会处于闲置状态，但是我们对COVID-19的工作表现出了极大的热情，以至于我们加紧展开更多模拟时，您会发现间歇性的停机时间。请耐心等待我们！有许多有价值的科学实验要做，而且我们正在尽可能快地使它运行。"
+
+msgid "Don’t miss our client FAQ. [HERE](https://foldingforum.org/viewtopic.php?f=24&t=32463)"
+msgstr "不要错我们的常见问题解答。[点击这里](https://foldingforum.org/viewtopic.php?f=24&t=32463)"
+
+msgid "To focus on Covid19 research set your category to any."
+msgstr "为了专注于Covid19研究，请将您的类别设置为任意。"

```

Comment:
There was an update to the file very recently. Feel free to add that to this PR, if you want to get that change in here as well. Can also be added later of course.

Reply:
Thanks! May you please request changes in this PR so I can edit it (I not familiar with the functionality of pull request and do not know other methods to edit the PR )? Thank you.

Reply:
Sorry, I didn't noticed that you had already pushed this changes. I have already approved about my review.